react/src/lib/Select/index.js

```diff
@@ -90,6 +90,7 @@ class Select extends React.Component {
 
   render() {
     const {
+      btnClassName,
```

Comment:
Should this be `buttonClassName`? Other than the old AngularJS code, we don't abbreviate button in our components.

Reply:
We have @pauljeter:
- Alerts:   dismissBtnProps: PropTypes.object,
- ModalHeader: closeBtnProps: PropTypes.object,

After our discussion, buttonProps wouldn't necessarily work since we are already applying a className to the button.  Can discuss in detail tomorrow.